render/render.go

```diff
@@ -11,6 +11,18 @@ import (
 type Renderer interface {
 	Render(report.Report) RenderableNodes
 	EdgeMetadata(rpt report.Report, localID, remoteID string) report.EdgeMetadata
+	Stats(report.Report) Stats
+}
```

Comment:
per f2f discussion: I was under the impression that the Nodes, Edges, and Stats were all returned via the same api call, so would continue travelling together.